plugins/utils/mini.nix

```diff
@@ -11,6 +11,9 @@ lib.nixvim.neovim-plugin.mkNeovimPlugin {
   package = "mini-nvim";
 
   extraOptions = {
+    mockDevIcons = lib.mkEnableOption "mockDevIcons" // {
+      description = "Whether to tell `mini.icons` to emulate `nvim-web-devicons` for plugins that don't natively support it.";
```

Comment:
Should we explicitly mention that this can be used as an alternative to `plugins.web-devicons.enable`? E.g. as a footnote in another paragraph.

Also, I'm wondering if we should have:

```nix
defaultText = literalMD "`false` **NOTE**: This option is experimental and the default value may change without notice.";
```